patches/textarea-caret@3.1.0.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/HISTORY.md b/HISTORY.md
+deleted file mode 100644
+index 24f38c3a6b6d0fc5a2abcecde78a3d2dd808ea79..0000000000000000000000000000000000000000
```

Comment:
No reason to delete this.

Reply:
Not sure how that sneaked in. Removed those lines now.